providers/common/macs/build.info

```diff
@@ -1,4 +1,8 @@
-$COMMON=cmac_prov.c gmac_prov.c hmac_prov.c kmac_prov.c
+IF[{- !$disabled{cmac} -}]
+  $COMMON=cmac_prov.c gmac_prov.c hmac_prov.c kmac_prov.c
+ELSE
+  $COMMON=gmac_prov.c hmac_prov.c kmac_prov.c
+ENDIF
```

Comment:
This example might give you a better idea on how not to repeat things so much :wink:

```
  $COMMON=$COMMON cmac_prov.c
```

Reply:
Fixed.